mix.exs

```diff
@@ -16,6 +17,10 @@ defmodule OAuther.Mixfile do
   defp description,
     do: "Library to authenticate with OAuth 1.0 protocol."
 
+  defp deps do
+    [{:dialyxir, "~> 0.3", only: [:dev]}]
```

Comment:
We shan't use either of them, we should let contributors use `mix install` on one of them so they can call the relevant Mix task from anywhere without this project having to depend on such libraries. Specs have nothing to do with how you call dialyzer anyways :)